### DISCUSSION TITLE
Running OpenPNM in Jupyter web platform requires a pure Python wheel

### DISCUSSION THREAD

**Discussion by AnnaHerringANU** [PMEAL/OpenPNM#2955] [Q&A] [answered]
I'm trying to run openpnm on a jupyter web notebook, but when tryong to pip install, I get the following error:

> ValueError: Can't find a pure Python 3 wheel for 'pyamg'.
> See: https://pyodide.org/en/stable/usage/faq.html#why-can-t-micropip-find-a-pure-python-wheel-for-a-package

From the link, it appears that a *py3-none-any.whl should be included in the download distribution.

**Comment by ma-sadeghi** [comment, marked as answer]
It's because `pyamg` uses C extensions. They provide a solution [here](https://pyodide.org/en/stable/development/new-packages.html#new-packages).
